src/overlays/actors/ovl_Boss_03/z_boss_03.c

```diff
@@ -41,157 +269,2320 @@ const ActorInit Boss_03_InitVars = {
-static ColliderJntSphElementInit D_809E8F24[5] = {
+// The limbs referenced here are not used. The position of those spheres is done manually on Boss03_PostLimbDraw
+static ColliderJntSphElementInit sJntSphElementsInit2[] = {
+    {
+        {
+            ELEMTYPE_UNK3,
+            { 0xF7CFFFFF, 0x00, 0x08 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { GYORG_LIMB_HEAD, { { 0, 0, 0 }, 20 }, 100 },
+    },
     {
-        { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 2, { { 0, 0, 0 }, 20 }, 100 },
+        {
+            ELEMTYPE_UNK3,
+            { 0xF7CFFFFF, 0x00, 0x08 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { GYORG_LIMB_LOWER_TRUNK, { { 0, 0, 0 }, 20 }, 100 },
     },
     {
-        { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 5, { { 0, 0, 0 }, 20 }, 100 },
+        {
+            ELEMTYPE_UNK3,
+            { 0xF7CFFFFF, 0x00, 0x08 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { GYORG_LIMB_TAIL, { { 0, 0, 0 }, 70 }, 100 },
     },
     {
-        { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 6, { { 0, 0, 0 }, 70 }, 100 },
+        {
+            ELEMTYPE_UNK3,
+            { 0xF7CFFFFF, 0x00, 0x08 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { GYORG_LIMB_RIGHT_FIN_ROOT, { { 0, 0, 0 }, 70 }, 100 },
     },
     {
-        { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 7, { { 0, 0, 0 }, 70 }, 100 },
+        {
+            ELEMTYPE_UNK3,
+            { 0xF7CFFFFF, 0x00, 0x08 },
+            { 0xF7CFFFFF, 0x00, 0x00 },
+            TOUCH_ON | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_ON,
+        },
+        { GYORG_LIMB_UPPER_RIGHT_FIN, { { 0, 0, 0 }, 30 }, 100 },
     },
+};
+
+static ColliderJntSphInit sJntSphInit2 = {
     {
-        { ELEMTYPE_UNK3, { 0xF7CFFFFF, 0x00, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
-        { 8, { { 0, 0, 0 }, 30 }, 100 },
+        COLTYPE_METAL,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_PLAYER,
+        OC2_TYPE_1,
+        COLSHAPE_JNTSPH,
     },
+    ARRAY_COUNT(sJntSphElementsInit2),
+    sJntSphElementsInit2,
+};
+
+Vec3f sGyorgInitialPos = { 1216.0f, 140.0f, -1161.0f };
```

Comment:
I wonder if this and other positions can be written relative to the centre of the room in some way.

Reply:
Do you mean the center of the platform?
The central platform is perfectly align to (0, Y, 0), and the platform has a height of 440 (that's Player's Y when he is standing on the platform). So you want something relative to (0, 440, 0)?